modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,380 @@
+----
++
+.Example output
+[source,terminal]
+----
+Successfully packaged chart and saved it to: /data/dagray/git/dagrayvid/special-resource-operator/yaml-for-docs/chart/simple-kmod-0.0.1/simple-kmod-0.0.1.tgz
+----
+
+. Create a ConfigMap to store the chart files
++
+[source,terminal]
+----
+mkdir cm
+cp simple-kmod-0.0.1.tgz cm/simple-kmod-0.0.1.tgz
+helm repo index cm --url=cm://simple-kmod/simple-kmod-chart
+oc create namespace simple-kmod
+oc create cm simple-kmod-chart --from-file=cm/index.yaml --from-file=cm/simple-kmod-0.0.1.tgz -n simple-kmod
+----
+
+
+. To deploy the simple-kmod using the Helm chart we created in the ConfigMap, the following SpecialResource manifest can be used. The `spec.chart.repository.url` field tells SRO to look for the chart in a ConfigMap. Optionally, uncomment the #debug: true line, to have the YAML files in the chart printed in full in the operator logs, to verify they are being created and templated properly. Save this YAML as `simple-kmod-configmap.yaml`
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: SpecialResource
+metadata:
+  name: simple-kmod
+spec:
+  #debug: true
+  namespace: simple-kmod
+  chart:
+    name: simple-kmod
+    version: 0.0.1
+    repository:
+      name: example
+      url: cm://simple-kmod/simple-kmod-chart
+  set:
+    kind: Values
+    apiVersion: sro.openshift.io/v1beta1
+    kmodNames: ["simple-kmod", "simple-procfs-kmod"]
+    buildArgs:
+    - name: "KMODVER"
+      value: "SRO"
+  driverContainer:
+    source:
+      git:
+        ref: "master"
+        uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
+----
+
+. Create the SpecialResource with:
++
+[source,terminal]
+----
+$ oc create -f simple-kmod-configmap.yaml
+----
+
+. The simple-kmod resources will be deployed in the namespace `simple-kmod` as specified in the object manifest. After a short time, the build pod for the simple-kmod driver container should start running. After a few minutes, the build should complete and the driver container Pods should start Running.
++
+[source,terminal]
+----
+$ oc get pods -n simple-kmod
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                  READY   STATUS      RESTARTS   AGE
+simple-kmod-driver-build-12813789169ac0ee-1-build     0/1     Completed   0          7m12s
+simple-kmod-driver-container-12813789169ac0ee-mjsnh   1/1     Running     0          8m2s
+simple-kmod-driver-container-12813789169ac0ee-qtkff   1/1     Running     0          8m2s
+----
+
+. The logs of the simple-kmod driver container image build can be displayed by getting the logs of the build pod. For example, using the pod name from the previous command.
++
+[source,terminal]
+----
+$ oc logs pod/simple-kmod-driver-build-12813789169ac0ee-1-build -n simple-kmod
+----
+
+. To verify that the simple-kmod kernel modules are loaded, execute the `lsmod` command in one of the driver container pods from step 3. 
```

Comment:
In the interest of long-term maintenance,  think we might be able to skip calling out the step number. Maybe just:

To verify that the simple-kmod kernel modules are loaded, execute the `lsmod` command in one of the driver container pods.